electrum/i18n.py

```diff
@@ -41,6 +43,11 @@ def set_language(x):
         language = gettext.translation('electrum', LOCALE_DIR, fallback=True, languages=[x])
 
 
+def get_default_language():
+    system_locale = QLocale.system().name()
+    return languages.get(system_locale, 'en_UK')
```

Comment:
Wouldn't this return something like `"German"` if `system_locale` was `de_DE`?

I think something like 

```python
return system_locale if system_locale in languages else "en_UK"
```

would make more sense.

Reply:
Oops, yeah, you right!

Reply:
I made this fix and moved this function to gui.qt.utils.